spec/gzip_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "logger"
+
+RSpec.describe HourlyLoggerRotator do
+  next if RUBY_VERSION.start_with?("2.2.")
+
+  context "gzip is not supported by OS" do
+    before do
+      allow(File).to(receive(:stat).with(/.*gzip/) { raise Errno::ENOENT, "test" })
+    end
+
+    it "raises an Error" do
+      expect { HourlyLoggerRotator.gzip=(true) }.to raise_error(RuntimeError)
```

Comment:
Would be really cool to check the error message so that we are sure that this is not some random error.

Reply:
ok